K8s small editorial changes (#1505)
RedisLabs/redislabs-docs: content/platforms/kubernetes/tasks/ldap-on-k8s.md

   ldapadd -h localhost -p 3889 -x -W -D "cn=admin,dc=example,dc=org" -f user.ldif
   ```

> ## Deploying a cluster with LDAP support

Reviewer: ```suggestion
## Deploy a cluster with LDAP support
```

### Deploying a Redis Enterprise cluster

> A Redis Enterprise cluster (REC) requires no special setup other to be configured with LDAP as the configuration is outside of the scope of the Redis Enterprise custom resource.

Reviewer: ```suggestion
A Redis Enterprise cluster (REC) requires no special setup other than to be configured with LDAP since the configuration is outside of the scope of the Redis Enterprise custom resource.
```

ldap_password: admin
```

> ### Using `rladmin` to update the configuration

Reviewer: ```suggestion
### Use `rladmin` to update the configuration
```
